NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 ## Breaking changes
 
+* `renderDataTable()`/`dataTableOutput()` have been officially deprecated and will automatically dispatch to their `{DT}` equivalents (`renderDT()`/`DTOutput()`) when a recent version of `{DT}` is available. If this happens to degrade an existing app, set `options(shiny.legacy.datatable = TRUE)` to get the old implementation. (#3998)
```

Comment:
Perhaps make it clearer in the first sentence that people should switch to the DT equivalents?